aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -1689,5 +1689,71 @@ Tensor chain_matmul(TensorList matrices) {
   }
 }
 
+/*
+Calculates the Kronecker product between two Tensors.
+*/
+Tensor kron(const Tensor& self, const Tensor& other) {
+  /*
+  We can obtain the kron result using tensordot or einsum.
```

Comment:
The below just uses tensordot, so maybe this should drop the "or einsum"?

Reply:
It may help understanding for people not familiar with tensordot but familiar with einsum and Einstein notation (I think einsum is more common). And the next sentence uses einsum.

Reply:
How about something like "using tensordot or einsum. The approach below uses tensordot."?